packages/checkout/widgets-lib/src/widgets/swap/components/SwapForm.tsx

```diff
@@ -151,6 +154,10 @@ export function SwapForm({ data, theme }: SwapFromProps) {
   const [gasFeeToken, setGasFeeToken] = useState<TokenInfo | undefined>(undefined);
   const [gasFeeFiatValue, setGasFeeFiatValue] = useState<string>('');
   const [tokensOptionsFrom, setTokensOptionsForm] = useState<CoinSelectorOptionProps[]>([]);
+  const formattedFees = useMemo(
+    (): any => (quote ? formatSwapFees(quote, cryptoFiatState, t) : []),
```

Comment:
@dreamoftrees Can we remove this `any`? Doesn't look like it will cause type issues if we do.